.storybook/preview.tsx

```diff
@@ -1,12 +1,13 @@
+import * as React from "react";
```

Comment:
why do you need to import react here?

Reply:
nvm i see it's for the IntlProvider

Reply:
wait is it? it wasn't needed before?

Reply:
its because its a component being returned not because of the IntlProvider, it was complaining that it didnt know what that jsx syntax was without the import

Reply:
linter is a more strict also